docs/custom_operations.md

```diff
@@ -148,7 +148,9 @@ is currently no way to handle it.
 Starting with the 20.07 release of Triton the [TorchVision
 operations](https://github.com/pytorch/vision) will be included with
 the PyTorch backend and hence they do not have to be explicitly added
-as custom operations.
+as custom operations. The Pytorch shared libraries must be available
```

Comment:
If the user does this they will break their tritonserver in the way we are seeing now. Did you try use --no-as-needed and linking torchvision against libtriton_pytorch.so?

Reply:
We already do that in our current "main" [here](https://github.com/triton-inference-server/pytorch_backend/blob/main/CMakeLists.txt#L129)

Reply:
Looking into alternative solutions as well

Reply:
To clarify: We do not need torchvision.so to load and run the torchvision custom ops. Torchvision looks for torch.so (and other pytorch deps) to load the ops/models.